plugin/pkg/admission/resourcequota/resource_access.go

```diff
@@ -63,7 +59,7 @@ type quotaAccessor struct {
 }
 
 // newQuotaAccessor creates an object that conforms to the QuotaAccessor interface to be used to retrieve quota objects.
-func newQuotaAccessor(client clientset.Interface) (*quotaAccessor, error) {
+func newQuotaAccessor() (*quotaAccessor, error) {
```

Comment:
It's hard because you get the client from 1 "Wants" interface and you get the informer factory from another.